backend/game_flow/lobby_repository.py

```diff
@@ -1,5 +1,5 @@
 from typing import Dict, Optional
-from backend.game_flow.game_pojos import *
+from game_flow.game_pojos import *
```

Comment:
Note on these changes: backend is not a module according to pyright. I refactored the imports here and there to make pyright stop complaining. Should have no impact as long as project is ran from the backend directory